CaloMC/src/CaloShowerROMaker_module.cc

```diff
@@ -226,11 +243,12 @@ namespace mu2e {
           {
               const CaloShowerStep& step = *istep;
 
-              // time folding and filtering, see docdb-3425 for a stunning explanation
+              // Time folding and filtering, see doc-db 3425 + talks from D. Brown for explanation
               double hitTimeUnfolded = toff_.totalTimeOffset(istep->simParticle())+istep->time();
-              double hitTime         = fmod(hitTimeUnfolded,mbtime_);
-
-              if (hitTime < blindTime_ || hitTime > mbtime_ ) continue;
+              double mbLength        = (ewMarker.spillType() == EventWindowMarker::SpillType::onspill) ? mbtime_ : ewMarker.eventLength();
+              double hitTime         = fmod(hitTimeUnfolded + pbtmc.pbtime_ + mbLength, mbLength);
```

Comment:
I think all the other systems have been doing no folding at all for offpsill events

Reply:
That seems right.  It's a tiny effect and I doubt that we will ever care about it.  Just drop any hits/steps at times outside of the window.  Should we ever decide to care about this we will need a coordinated discussion that involves all subsystems plus whoever is defining mixing-streams for off spill.